### PR TITLE
Add lsp-jedi to language index

### PR DESCRIPTION
**Pull request by wyuenho**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -70,6 +70,7 @@ nav:
     - PHP (felixbecker): page/lsp-php.md
     - Powershell: page/lsp-pwsh.md
     - PureScript: page/lsp-purescript.md
+    - Python: page/lsp-jedi.md
```
Comment (ericdallo) [inline mkdocs.yml:73]: Actually @wyuenho, could you add some sufix to the key? something like: 
```suggestion
    - Python (jedi): page/lsp-jedi.md
```

Comment (wyuenho) [reply mkdocs.yml:73]: I feel like we should add (Palantir) to pyls instead, and steer people to start with lsp-jedi first. Pyls is so slow that almost no one uses it anymore. What do you think?

Comment (ericdallo) [reply mkdocs.yml:73]: I see no problem with that, but I'm not a heavy python user, maybe @mpanarin could confirm that?

Comment (mpanarin) [reply mkdocs.yml:73]: Are we really gonna argue about what should go first? :\
As well as saying `almost no one uses it anymore` nothing else but a stretch...

Anyway, I am fine with adding `(Palantir)` to pyls and `(jedi)` to this ls.

Comment (ericdallo) [reply mkdocs.yml]: @wyuenho, Could you rename Pyls and jedi so?

Comment (wyuenho) [reply mkdocs.yml]: At risk of being argumentative, I must say @mpanarin you are the only one being argumentative here.

**Comment by ericdallo** [comment]
Thank you!

**Pull request merged**